OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -366,6 +366,7 @@
     <Compile Include="Traits\ProximityCapturable.cs" />
     <Compile Include="Traits\RadarColorFromTerrain.cs" />
     <Compile Include="Traits\Render\Hovers.cs" />
+    <Compile Include="Traits\Render\LeavesTrack.cs" />
```

Comment:
This is kind of nitpicky, but I think `LeavesTrail` would be better, as this can (potentially) be used for infantry, tracked and wheeled vehicles, hovercraft and sandworms.